automation/build-bin.ts

```diff
@@ -151,6 +153,69 @@ async function buildPkg() {
 	);
 }
 
+/**
+ * Run some basic tests on the built pkg executable.
+ */
+async function testPkg() {
```

Comment:
So far we didn't have any automated testing for standalone zip packages. This function tests only `balena version -j` (comparing the output against `process.version`, to ensure that the native modules were compiled with a compatible version of Node.js). In the future, we should probably look at somehow integrating some of the the CLI source code / command tests (mocha, chai) so that they also test the built pkg executable.